src/lib.rs

```diff
@@ -597,7 +605,14 @@ fn readline_edit<H: Helper>(
                 s.refresh_line()?;
                 continue;
             }
-            Cmd::Noop | _ => {
+            #[cfg(not(unix))]
+            Cmd::Suspend => {}
+            Cmd::Move(Movement::WholeLine) => {} // not an actual movement
+            Cmd::Complete | Cmd::CompleteBackward | Cmd::CompleteHint |
+            Cmd::ForwardSearchHistory | Cmd::ReverseSearchHistory |
+            Cmd::Abort | Cmd::Insert(_, _) | Cmd::SelfInsert(_, _) |
+            Cmd::Noop | Cmd::Unknown
+            => {
```

Comment:
Is this change related to https://rust-lang.github.io/rust-clippy/master/index.html#wildcard_in_or_patterns ?
Why not just replace `Cmd::Noop | _` by `_` ?

Reply:
This way it's much easier to add a new command. The compiler shows all the places where a new command should be added, including this match statement. And no it's not to satisfy Clippy.

I can revert this change if you don't think it makes sense.

Reply:
If you don't mind, I would prefer that you revert this change.
Thanks.

Reply:
Reverted. Sorry for long delay. (I've reverted in a different commit, I expect you to squash changes when merging. Let me know if you want me to squash changes.